proguard/proguard-square-retrofit2.pro

```diff
@@ -0,0 +1,11 @@
+# Retrofit 2.X
+## https://square.github.io/retrofit/ ##
+
+-dontwarn retrofit2.**
+-keep class retrofit2.** { *; }
+-keepattributes Signature
+-keepattributes Exceptions
+
+-keepclasseswithmembers class * {
+    @retrofit2.http.* <methods>;
+}
```

Comment:
maybe more shrinks official proguard 🚋 

https://github.com/square/retrofit#proguard

Reply:
It seems to be still under discussion. 🤔 
ref: https://github.com/square/retrofit/pull/2338

In my app, the following is required but keeping `@retrofit2.http` is not needed 🤔 

```
-dontnote retrofit2.Platform
-dontwarn retrofit2.Platform$Java8
```

Reply:
As they commented, web-site and README don't have consistency. 🤔

Reply:
🤔 

i looks like only under rules 💭 

```
-dontnote retrofit2.Platform
-dontwarn retrofit2.Platform$Java8
```